src/district/ui/smart_contracts/events.cljs

```diff
@@ -87,6 +104,17 @@
           :on-success [::contract-loaded contract-to-load true *load-batch*]
           :on-failure [::contract-loaded contract-to-load false *load-batch*]})})))
 
+(reg-event-fx
+ ::use-loaded-contracts
```

Comment:
use our ui logging facilities for some error handling if this fails: https://github.com/district0x/district-ui-logging

Reply:
Added district-ui-logging